### PR TITLE
*: add ci for images management and greeting.

### LINKED ISSUE #134: [feature] add workflow and Travis CI
**Is your feature request related to a problem? Please describe.**
Add workflow
Add Travis CI

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by runkecheng**
### What type of PR is this?

/enhancement

### Which issue(s) this PR fixes?

Fixes #134

### What this PR does?

Summary:

- Automatically update the corresponding images(operator, sidecar) according to the modified file.
- Automatically greet when submitting an issue for the first time.

### Special notes for your reviewer?

Also need to add:

- Automatic test when submitting pr
- Automatically add tags to issue/pr

**To make the yml file effective, must set the environment variables first!**

### CODE REVIEW

**Review by acekingke — COMMENTED** [review]
LGTM

File: .github/workflows/auto_build_operator.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    paths:
+     - cmd/manager/*
+     - api/*
+     - cluster/*
+     - controllers/*
+     - internal/*
+     - utils/*
+    
+jobs:
+
```
Comment (andyli029) [inline .github/workflows/auto_build_operator.yml:15]: `15` `17`
Delete the space line.

File: .github/workflows/auto_build_operator.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: auto build operator
```
Comment (hustjieke) [inline .github/workflows/auto_build_operator.yml:1]: change `operator` to `ctlmanager`will be better ? 
I think there will be more controllers in our mysql operator and here we make build the controllers.

Comment (runkecheng) [reply .github/workflows/auto_build_operator.yml:1]: The name of our image is called mysql-operator, so I chose this name. whick one is  better? @andyli029

**Comment by hustjieke** [comment]
LGTM.

**Pull request merged**